### ISSUE TITLE
JetSmartFilters + SearchWP integration

### ISSUE THREAD

**Issue by poziomapl** [Crocoblock/suggestions#810]
It would be great if JetSmartFilters plugin (Search element) will work with powerful SearchWP.

SearchWP allows to display specific search results - not only for post titles and its content, but also for taxonomy or your many own custom fields and more.
In addition, it allows searching not only after whole phrases such as "_London_", but also partial matches, such as "_Lon_".

Just like in Smart Filters Settings it is possible to select the "Default WordPress search" and "by "Custom Fiels (by Query Variable)" so there could also be a field to choose SearchWP.

For now, I have to use FacetWP because it is integrated with SearchWP. However, then I can't use Listing Grid from JetEngine only Elementor Pro Posts, because JetEngine is not compatible with FacetWP.

Therefore, it would be great if JetEngine would work with FacetWP if someone would consider it an alternative to JetSmartFilters.

**Comment by realstephenzhao** [comment]
I think you'd better suggest JetSearch to be a better version of SearchWP.  Using SearchWP and FacetWP will cost much more money and harm the performance of your website.

**Comment by vloot65** [comment]
I agree that it would be great to have JetSmartFilters be able to use SearchWP (or to use the default SearchWP engine as 'default wordpress search'. To be honest I assumed it would work that way, that is why I bought both plugins.

**Comment by poziomapl** [comment]
I think i found a solution: 

Integrate SearchWP with JetSmartFilters search using JetEngine Listing Grid to display results

https://gist.github.com/jchristopher/39cf1f628dd678fabc64021ee5dc503b
